### DISCUSSION TITLE
Snap Areas for Super Ultra Wide Monitors

### DISCUSSION THREAD

**Discussion by arlaneenalra** [rxhanson/Rectangle#776] [Ideas]
I have a monitor with a 32x9 aspect ratio. It's effectively the same as having 2 16x9 monitors without the bezel. It would be extremely useful to be able to define/enable 1/4 drag/snap areas instead of having to use keyboard shortcuts.

**Comment by rxhanson** [comment]
Thanks for the feedback. I don't quite have the bandwidth to make these adjustable at the moment, but see no reason to turn down a good PR for it.
